src/Documentation/sidebar.json

```diff
@@ -120,6 +120,80 @@
         "label": "Managing External Data",
         "slug": "managing-external-data"
       },
+      {
+        "label": "Remotes",
+        "slug": "remotes",
+        "source": "remotes/index.md",
+        "children": [
+          {
+            "label": "Local",
+            "slug": "local"
+          },
+          {
+            "label": "HTTP",
+            "slug": "http"
+          },
+          {
+            "label": "SSH",
+            "slug": "ssh"
+          },
+          {
+            "label": "Amazon S3",
+            "slug": "amazon"
+          },
+          {
+            "label": "S3 Compatible",
+            "slug": "s3"
+          },
+          {
+            "label": "Azure",
+            "slug": "azure"
+          },
+          {
+            "label": "Google Cloud Storage",
+            "slug": "gs"
+          },
+          {
+            "label": "HDFS",
+            "slug": "hdfs"
+          },
+          {
+            "label": "Aliyun OSS",
+            "slug": "oss"
+          }
+        ]
+      },
+      {
+        "label": "External Data",
```

Comment:
why can't we keep the Managing External Data label and name? Not much difference but redirect won't be required.

Reply:
> why can't we keep the Managing External Data label and name? Not much difference but redirect won't be required.

Redirect would still be required because of "external-dependencies".
On the other hand "External Data" is more generic than "Managing External Data". For example in the case of HTTP (and other cases) we are just accessing or downloading external data, "managing" doesn't seem suitable.

Reply:
Again, please, don't resolve. I have to repeat myself and create duplicates.

Don't see any reasons to rename and complicate this.

Reply:
> Don't see any reasons to rename and complicate this.

I don't understand, what is the complication that comes from renaming? In the first comment you say "not much difference", and I agree with this. And redirect is still required even if we don't change the name.

Reply:
one extra redirect to support + updating links in the docs

Reply:
I have already updated the links on the docs, and also have fixed the redirect.
Changing the name back would be extra work, and unjustified, unless the old name is better than the new one.